### PR TITLE
Update circleci-docker-primary image to point to most recent

### PR DESCRIPTION
**Pull request by LeDeep**
Points to the most recent [`trussworks/circleci-docker-primary`](https://github.com/trussworks/circleci-docker-primary/commit/ba270cf66c7a9ae6dbc8e6190b74163ad71729d0).

### CODE REVIEW

**Comment by chrisgilmerproj** [comment]
Did you run `make e2e_test_docker` locally? Just want to make sure that all the `Dockerfile.*local` files work with the new image.

**Comment by LeDeep** [comment]
> Did you run `make e2e_test_docker` locally? Just want to make sure that all the `Dockerfile.*local` files work with the new image.

I did. All green.

**Comment by LeDeep** [comment]
> Looks like we need to update this too:
> https://github.com/transcom/mymove/blob/dfb02ec0bc6d97bfe752cff8b4f3571552694948/.github/workflows/go-auto-approve.yml#L28

nice catch! pushed update

**Comment by reggieriser** [comment]
I don't see any issues with this if all the tests pass, but you may want an approval from infra too before merging.

File: .github/workflows/go-auto-approve.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
       - name: setup go
         uses: actions/setup-go@v1
         with:
-          go-version: '1.13.4'
+          go-version: '1.13.7'
```
Comment (chrisgilmerproj) [inline .github/workflows/go-auto-approve.yml]: I think we have docs about updating the docker image. We may want to point out in the docs that this also needs updating.

Comment (LeDeep) [reply .github/workflows/go-auto-approve.yml]: It was actually already in the documentation. I moved into the `Update transcom/mymove repo` section. I also updated the documentation to account for the new files that will need to be updated. 

The `transcom/ppp` circleci config passes in `latest` for the docker image. So, I removed that section from the readme as those steps don't seem relevant anymore.

**Pull request merged**